### ISSUE TITLE
Fix _pydoctor.py to allow the case when intersphinx API docs could not be fetched

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8604]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#8604|
|Type|defect|
|Created|2016-07-15 17:05:23Z|
|Branch|https://github.com/twisted/twisted/tree/8604-rodrigc-zope-pydoctor|

see this error 

https://buildbot.twistedmatrix.com/builders/documentation/builds/900/steps/api-documentation/logs/stdio

.... if zope API website is down, zope api is not loaded and then

```
self._links.get('zope.interface.interfaces.IInterface')
```

will return None

now... pydoctor might report a warning when a remote intershinx could not be loaded

<details><summary>Searchable metadata</summary>

```
trac-id__8604 8604
type__defect defect
reporter__adiroiban adiroiban
priority__normal normal
milestone__None None
branch__8604_rodrigc_zope_pydoctor 8604-rodrigc-zope-pydoctor
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1468602323501284 1468602323501284
changetime__1474198049187875 1474198049187875
version__None None
owner__hawkowl hawkowl

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban set owner to @hawkowl|
|-|-|

Amber told me over IRC that she is working on this.

For the pydoctor part I have fixed the error logging here https://github.com/twisted/pydoctor/pull/118

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_2"></a>|@hawkowl commented|
|-|-|

I am working on some other stuff in _pydoctor.py, could you clarify what needs to be fixed in it so I can make sure it's included?

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban commented|
|-|-|

The patch is here https://gist.github.com/adiroiban/642850727d2e49937cb582a3fd284cbd#file-twisted-diff-L3

only twisted/python/_pydoctor.py

The problem is that pydoctor expects that zope.interface intersphinx links are always loaded... but if the zope.interface docs website is done, we get this ugly error.

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_4"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/372

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|GitHub set status to `closed`|
|-|-|

In changeset edcf16c3c61f619c312b1934972d1675bb08d426
```
#!CommitTicketReference repository="" revision="edcf16c3c61f619c312b1934972d1675bb08d426"
Merge pull request #372 from twisted/8604-rodrigc-zope-pydoctor

Author: rodrigc
Reviewer: hawkowl
Fixes: #8604
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
